src/Microsoft.SqlTools.ServiceLayer/Agent/Contracts/AgentJobHistoryInfo.cs

```diff
@@ -8,6 +8,7 @@
 using System.Collections.Generic;
 using Microsoft.SqlServer.Management.Common;
 using Microsoft.SqlTools.ServiceLayer.Agent;
+using Microsoft.SqlServer.Management.Smo.Agent;
```

Comment:
why are you adding this include?  we shouldn't be sending any SMO objects over the JSON-RPC channel.